src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -969,20 +983,28 @@ public static int getInventoryScale() {
         return inventoryScale == 0 ? -1 : inventoryScale;
     }
 
-    @Switch(
-        name = "Remove Container Background",
+//    @Switch(
+//        name = "Remove Container Background",
+//        description = "Remove the dark background inside a container.",
+//        category = "Screens", subcategory = "General"
+//    )
+    public static boolean removeContainerBackground = false;
+
+    @Slider(
+        name = "Change Container Background Opacity (%)",
         description = "Remove the dark background inside a container.",
-        category = "Screens", subcategory = "General"
+        category = "Screens", subcategory = "General",
+        min = 0F, max = 100F
     )
-    public static boolean removeContainerBackground = false;
+    public static float containerBackgroundOpacity = (208/255F) * 100F;
```

Comment:
add a thing at the patcher constructor to change the opacity to 0 if removeContainerBackground used to be on

Reply:
did it i think, lmk if it isn't right